virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -224,6 +224,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 * If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might also fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2055595*])
 ** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
 
+//CNV-33789: Cannot SSH into VM over NodePort and Console's FQDN when using OVNKubernetes networking
+* You cannot SSH into a VM when using the `networkType: OVNKubernetes` option in your `install-config.yaml` file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2165895)[*BZ#2165895*])
```

Comment:
```suggestion
* You cannot SSH into a VM when using the `networkType: OVNKubernetes` option in your `install-config.yaml` file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2165895[*BZ#2165895*])
```

Reply:
thanks for spotting that! and sorry i forgot the preview link. edited the OP to include it :)

Reply:
no worries! lots going on this week :)